demos/tests/args.py

```diff
@@ -52,6 +52,15 @@ def resolve(self, context):
         return str(context.dl_dir / context.model_info[self.name]["subdirectory"] / self.precision / (self.name + '.xml'))
 
 
+class ModelOnnxArg:
```

Comment:
This seems needlessly specific; I suggest that you wait until #2291 is merged and reuse the argument class that is added there.

Reply:
Okay